src/components/Moviepage/MoviePage.jsx

```diff
@@ -1,5 +1,69 @@
-import React from "react";
+import axios from "axios";
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import Spinner from "../Spinner/Spinner";
 
-export default function MoviePage({ movieInfo }) {
-  return <div>{movieInfo.id}</div>;
+const api_key = process.env.REACT_APP_API_KEY;
+
+export default function MoviePage({ isLoading, setIsLoading }) {
+  const [movieInfo, setMovieInfo] = useState([]);
+  const params = useParams();
+
+  useEffect(() => {
+    fetchMovieData(params.id);
+  }, []);
+
+  async function fetchMovieData(id) {
+    setIsLoading(true);
+    try {
+      const response = await axios.get(
+        `https://api.themoviedb.org/3/movie/${id}`,
+        {
+          params: {
+            api_key: api_key,
+          },
+        }
+      );
+      setMovieInfo(response.data);
+      setIsLoading(false);
+    } catch (error) {
+      console.log(error);
+    }
+  }
+
+  return (
+    <>
+      {isLoading ? (
+        <Spinner />
+      ) : (
+        <section>
```

Comment:
In my opinion` <article/>` html tag suits better. Because `<article/>` specifies independent, self-contained content. Whereas `<section />  `defines only a section in a document